### PR TITLE
Small clarifications to README

### PR DESCRIPTION
**Pull request by nathanl**
Also, I deliberately added the phrase "recurring job" to the readme here for searchability. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -562,6 +562,21 @@ The crontab format respects all [standard rules][cron] and has one minute
 resolution. Jobs are considered unique for most of each minute, which prevents
 duplicate jobs with multiple nodes and across node restarts.
 
+Workers which will be used in recurring jobs do not need to specify a queue.
```
Comment (sorentwo) [inline README.md:565]: This isn't accurate. If no queue is specified then they will use the `default` queue, just like any other job.

File: README.md [hunk 2]
```diff
@@ -562,6 +562,21 @@ The crontab format respects all [standard rules][cron] and has one minute
...
+
+```elixir
+defmodule MyApp.PeriodicJob do
+  use Oban.Worker # no queue specified
+
+  @impl Oban.Worker
+  def perform(_args, _job) do
```
Comment (sorentwo) [inline README.md:572]:
```suggestion
  def perform(_job) do
```

File: README.md (hunk 1 above)
Comment (nathanl) [reply README.md:565]: Ah, ok, thanks.

**Comment by sorentwo** [comment]
Thanks!

**Pull request merged**